src/main/java/org/jenkinsci/test/acceptance/utils/FormElementPath.java

```diff
@@ -0,0 +1,95 @@
+package org.jenkinsci.test.acceptance.utils;
+
+import org.apache.http.HttpEntity;
+import org.apache.http.HttpResponse;
+import org.apache.http.NameValuePair;
+import org.apache.http.auth.Credentials;
+import org.apache.http.client.entity.UrlEncodedFormEntity;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.ContentType;
+import org.apache.http.entity.mime.HttpMultipartMode;
+import org.apache.http.entity.mime.MultipartEntityBuilder;
+import org.apache.http.entity.mime.content.FileBody;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.message.BasicNameValuePair;
+import org.apache.http.util.EntityUtils;
+
+import javax.annotation.CheckForNull;
+import javax.annotation.Nonnull;
+import javax.inject.Inject;
+import javax.inject.Named;
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+
+/**
+ * Ensures form-element-path plugin is running, and if not, installs it using the rest api to upload a plugin
+ */
+public class FormElementPath {
+    private static final Logger LOGGER = Logger.getLogger(FormElementPath.class.getName());
+
+    private Credentials credentials;
+
+    @Inject
+    @Named("form-element-path.hpi")
+    private File formElementPathPlugin;
```

Comment:
Could actually be put into the ATH classpath via `dependency:copy`.

Reply:
This was already in place, I figured this could just be reused.